docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -277,6 +277,13 @@ type:: string
 default:: `keycloak`
 
 
+== `database.jdbcParams`
+
+[horizontal]
+type:: string
+default:: `sslmode=verify-ca&sslrootcert=/opt/jboss/certs/tls.crt`
+
```

Comment:
Probably worth it to make a note that the default parameters **must** remain set when DB TLS is enabled.